l10n_ch_hr_payroll/data/hr.salary.rule.xml

```diff
@@ -0,0 +1,531 @@
+            <field name="appears_on_payslip">False</field>
+            <field name="code">P13_PCF</field>
+            <field name="condition_select">none</field>
+            <field name="name">13ème PCF</field>
+            <field name="amount_python_compute">result = PCF_C/12</field>
+            <field name="quantity">1</field>
+            <field name="sequence">290</field>
+            <field name="category_id" ref="hr_payroll_P13"/>
+            <field name="parent_rule_id" ref="P13"/>
+        </record>
+        <record id="P13_FADMIN" model="hr.salary.rule">
+            <field name="amount_select">code</field>
+            <field name="appears_on_payslip">False</field>
+            <field name="code">P13_FADMIN</field>
+            <field name="condition_select">none</field>
+            <field name="name">13ème FADMIN</field>
+            <field name="amount_python_compute">result = FADMIN/12</field>
+            <field name="quantity">1</field>
+            <field name="sequence">295</field>
+            <field name="category_id" ref="hr_payroll_P13"/>
+            <field name="parent_rule_id" ref="P13"/>
+        </record>
+        <record id="I_VAC" model="hr.salary.rule">
+            <field name="amount_select">code</field>
+            <field name="appears_on_payslip">False</field>
+            <field name="code">I_VAC</field>
+            <field name="condition_select">python</field>
+            <field name="name">Indemnité vacances</field>
+            <field name="amount_python_compute">result = (contract.wage * contract.holiday_rate) / 100</field>
+            <field name="condition_python">result = contract.holiday_rate > 0</field>
+            <field name="quantity">1</field>
+            <field name="sequence">25</field>
+            <field name="category_id" ref="hr_payroll.ALW"/>
+        </record>
+    <!-- </data> -->
+    <!-- <data> -->
+        <record id="hr_payroll.hr_rule_taxable" model="hr.salary.rule">
+            <field name="amount_select">code</field>
+            <field name="appears_on_payslip">True</field>
+            <field name="code">GROSS</field>
+            <field name="condition_select">none</field>
+            <field name="name">Gross</field>
+            <field name="amount_python_compute">result = categories.BASIC + categories.ALW + categories.FALW</field>
+            <field name="condition_python"> # Available variables: #---------------------- # payslip: object containing the payslips # employee: hr.employee object # contract: hr.contract object # rules: object containing the rules code (previously computed) # categories: object containing the computed salary rule categories (sum of amount of all rules belonging to that category). # worked_days: object containing the computed worked days # inputs: object containing the computed inputs  # Note: returned value have to be set in the variable 'result'  result = rules.NET > categories.NET * 0.10</field>
```

Comment:
Shouldn't you add line break of some kind in this line. Isn't this a one line string ?